process/elasticproc/logsevents/delegatorsProcessor.go

```diff
@@ -85,10 +90,26 @@ func (dp *delegatorsProc) processEvent(args *argsProcessEvent) argOutputProcessE
 		Contract:       contractAddr,
 		ActiveStake:    activeStake.String(),
 		ActiveStakeNum: dp.balanceConverter.ComputeBalanceAsFloat(activeStake),
+		Timestamp:      time.Duration(args.timestamp),
 	}
 
 	if eventIdentifierStr == withdrawFunc && len(topics) >= minNumTopicsDelegators+1 {
 		delegator.ShouldDelete = bytesToBool(topics[4])
+		withdrawFundIds := topics[5:]
```

Comment:
this can panic, you didn't changed the `minNumTopicsDelegators` value

Reply:
it will not panic. 
If the topics slice will have 5 elements then `withdrawFundIds` will be nil but  it will not panic